Dockerfile

```diff
@@ -9,6 +9,7 @@ HEALTHCHECK --interval=5m --start-period=2m \
 USER root
 
 # Install tools
+# hadolint ignore=DL3008
```

Comment:
Why?

Ignore comments should never be used.

Reply:
As discussed, package versions will be specified.